Assets/Scripts/Layers/LayerTypes/HierarchicalObjectLayer.cs

```diff
@@ -50,20 +50,25 @@ private void OnMouseClickNothing()
         
         public void OnPointerClick(PointerEventData eventData)
         {
+            if(ReferencedProxy.UI == null) return;
+
             ReferencedProxy.UI.Select(true);
         }
         
         public override void OnSelect()
         {
-            var rth = FindAnyObjectByType<RuntimeTransformHandle>(FindObjectsInactive.Include); //todo remove FindObjectOfType
-            rth.SetTarget(gameObject);
+            var transformHandle = FindAnyObjectByType<RuntimeTransformHandle>(FindObjectsInactive.Include); //todo remove FindObjectOfType
+            
+            if(transformHandle)
```

Comment:
dat is vreemd als dit een probleem oplevert, die handles zouden altijd in de scene moeten zijn